modules/understanding-upgrade-channels.adoc

```diff
@@ -69,12 +69,14 @@ Do not rely on consecutive patch numbers. In this example, {product-version}.2 i
 ====
 
 [id="conditional-updates-overview_{context}"]
-== Update recommendation removals and Conditional Updates
-Red Hat monitors newly released versions and update paths associated with those versions before and after they are added to supported channels. If a serious regression is identified, Red Hat may remove affected update recommendations. When Red Hat chooses to remove update recommendations, that action is taken in all relevant channels simultaneously. The removal of recommended updates may happen either before or after updates have been promoted to supported channels.
+== Update recommendations and Conditional Updates
+Red Hat monitors newly released versions and update paths associated with those versions before and after they are added to supported channels.
 
 If Red Hat removes update recommendations from any supported release, a superseding update recommendation will be provided to a future version that corrects the regression. There may however be a delay while the defect is corrected, tested, and promoted to your selected channel.
 
-Beginning in {product-title} 4.10, when update recommendations are removed from supported channels, they are replaced with Conditional Updates that declare one or more known risks. Each known risk may apply to all clusters or only clusters matching certain conditions. Some examples include having the `Platform` set to `None` or the CNI provider set to `OpenShiftSDN`. The Cluster Version Operator (CVO) continually evaluates known risks against the current cluster state. If no risks match, the update is recommended. If the risk matches, those updates are listed as a `Supported But Not Recommended` update and a reference link is provided. The reference link helps the cluster admin decide if they would like to accept the risk and update anyway.
+Beginning in {product-title} 4.10, when update risks are discovered, they are declared as Conditional Update risks. Each known risk may apply to all clusters or only clusters matching certain conditions. Some examples include having the `Platform` set to `None` or the CNI provider set to `OpenShiftSDN`. The Cluster Version Operator (CVO) continually evaluates known risks against the current cluster state. If no risks match, the update is recommended. If the risk matches, those updates are listed as a `Supported But Not Recommended` update and a reference link is provided. The reference link helps the cluster admin decide if they would like to accept the risk and update anyway.
```

Comment:
Just minor suggestion on the wording, feel free to ignore the comment if they look not right:)

"Beginning in {product-title} 4.10, when update risks are discovered, they are declared as Conditional Update risks." -> "Beginning in {product-title} 4.10, when update risks are discovered and confirmed, they are declared as Conditional Update risks and applied to the relevant update paths."

"If no risks match, the update is recommended. If the risk matches, those updates are listed as a `Supported But Not Recommended` update and a reference link is provided. "-> 

"If no risks match, the update is listed as a `Recommended` update. If the risk matches, the update listed as a `Supported but not recommended` update and a reference link is provided."

Reply:
I've pushed 3197415fb0 -> 912557ea0e, rebasing onto `main` and rewording:

```console
$ git --no-pager diff -U0 --word-diff 3197415fb0..912557ea0e -- modules/understanding-upgrade-channels.adoc
diff --git a/modules/understanding-upgrade-channels.adoc b/modules/understanding-upgrade-channels.adoc
index f74a249e6d..51bf00d909 100644
--- a/modules/understanding-upgrade-channels.adoc
+++ b/modules/understanding-upgrade-channels.adoc
@@ -77 +77 @@ If Red Hat removes update recommendations from any supported release, a supersed
Beginning in {product-title} 4.10, when update risks are [-discovered,-]{+confirmed,+} they are declared as Conditional Update [-risks.-]{+risks for the relevant updates.+} Each known risk may apply to all clusters or only clusters matching certain conditions. Some examples include having the `Platform` set to `None` or the CNI provider set to `OpenShiftSDN`. The Cluster Version Operator (CVO) continually evaluates known risks against the current cluster state. If no risks match, the update is recommended. If the risk matches, those updates are [-listed as a `Supported But Not Recommended` update-]{+supported but not recommended,+} and a reference link is provided. The reference link helps the cluster admin decide if they would like to accept the risk and update anyway.
```

New paragraph in the updated commit message:

> I'm using "confirmed" from [Yang's suggested rewording][2], to hint that there is [an assessment process][3] between a potential-regression report and an update risk being declared.
>
> I'm using generic wording for "recommended" and "supported but not recommended" instead of trying to quote user interfaces, because `oc adm upgrade` and the in-cluster web console don't quite agree on wording:
>
> * Recommended:
>   * oc: "[Recommended updates][4]".
>   * web-console: "[Recommended][5]".
> * Supported but not recommended:
>   * oc: "[Supported but not recommended updates][6]".
>   * web-console: "[Include supported but not recommended versions][7]".

[2]: https://github.com/openshift/openshift-docs/pull/58771#discussion_r1189306533
[3]: https://github.com/openshift/enhancements/blob/ba306451b99da182d26bd0631f3f1e448930316c/enhancements/update/update-blocker-lifecycle/README.md
[4]: https://github.com/openshift/oc/blame/f843bb1e3acc667e60fd6831afbc6da5b0063518/pkg/cli/admin/upgrade/upgrade.go#LL417C26-L417C46
[5]: https://github.com/openshift/console/blob/07fe8898570666409b38e46630559da6e3adf25f/frontend/public/components/modals/cluster-update-modal.tsx#L178
[6]: https://github.com/openshift/oc/blame/f843bb1e3acc667e60fd6831afbc6da5b0063518/pkg/cli/admin/upgrade/upgrade.go#L441
[7]: https://github.com/openshift/console/blob/07fe8898570666409b38e46630559da6e3adf25f/frontend/public/components/modals/cluster-update-modal.tsx#L210